README.md

```diff
@@ -7,6 +7,8 @@
 
 Bus provides a lock-free, bounded, single-producer, multi-consumer, broadcast channel.
 
+> NOTE!: There is an issue in the current implementation which causes CPU usage during idle operation. See [#23](https://github.com/jonhoo/bus/issues/23). Consider using [tokio::sync::broadcast](https://docs.rs/tokio/0.3.5/tokio/sync/broadcast/) instead of this crate.
```

Comment:
Hmm, I don't think this `tokio::sync::broadcast` is a good alternative recommendation here, since it is explicitly for async use-cases, whereas bus is for synchronous use. I don't personally know of any synchronous bus alternatives though unfortunately ...

Reply:
All you need is `futures::executor::block_on(receiver.recv())` to get a sync (this is what I migrated to).

Reply:
That is a lot of additional dependencies to pull in as a replacement. Basically pull in lots of async mechanisms when you actually just want the synchronous implementation. I don't think I'd reasonably recommend that as a replacement. I think warning about the issue is sufficient.

Reply:
Great, feel free to put in any text you want. I'm just trying to help :)